client/jetpack-cloud/sections/partner-portal/payment-method-delete-dialog/index.tsx

```diff
@@ -1,8 +1,9 @@
 import { Button, Dialog } from '@automattic/components';
```

Comment:
We should disable the "Delete payment method" button until the next payment method query finishes so clicking the Delete button while the query is not yet resolved doesn't send a request without another primary method being passed along when one is available.

Reply:
Thanks! Fixed in 0aed5538d432fafba6335a497f2fb43d58a43ef1